src/api/auth.py

```diff
@@ -20,9 +19,9 @@
     summary='Login for token'
 )
 async def login(
-    form_data: OAuth2PasswordRequestForm = Depends(),
-    db: AsyncSession = Depends(get_db),
-    authorize: AuthJWT = Depends()
+        form_data: OAuth2PasswordRequestForm = Depends(),
```

Comment:
Po co ta zmiana w wcięciu?

Reply:
Zmiany we wcięciach i liniach odstępu wynikały z użycia ctrl+alt+l do automatycznego formatowania, poprawię żeby było spójnie z resztą